frontend/packages/dev-console/integration-tests/features/vulnerability/install_quay_container_security_operator

```diff
@@ -0,0 +1,20 @@
+Feature: Install quay container security operator
+    As a user I would like to see Image Vulnerability for each Project
+
+Background:
+    Given user is in Admin perspecitve
+
+@regression
+Scenario: Quay container security operator
+   Given user is at Operator Hub page with the header name "OperatorHub"
+   When user searches for "quay container security"
```

Comment:
```suggestion
   When user searches for "Quay Container Security"
```

Reply:
Can we have the operator name as it is on the Operator Hub?

Reply:
The name of the operator is in small caps in the operator-hub